kie-server-parent/kie-server-controller/kie-server-controller-api/src/main/java/org/kie/server/controller/api/model/spec/ProcessConfig.java

```diff
@@ -63,6 +69,10 @@ public void setRuntimeStrategy(String runtimeStrategy) {
         this.runtimeStrategy = runtimeStrategy;
     }
 
+    public void setRuntimeStrategy(RuntimeStrategy runtimeStrategy) {
+        this.runtimeStrategy = runtimeStrategy.name();
+    }
+
```

Comment:
Well I can check if `RuntimeStrategy.valueOf(runtimeStrategy)` throws an exception. If it doesn't, then I can assign the argument. Another possibility would be (since valueOf creates directly enum) to save runtime strategy as enum inside ProcessConfig class, but then it can have bigger impact on marshalling etc. So I see 2 options there...

Reply:
I'd go for option 1